libs/partners/upstage/langchain_upstage/layout_analysis_parsers.py

```diff
@@ -181,19 +181,22 @@ def _get_response(self, files: Dict) -> List:
 
             result = response.json().get("elements", [])
 
+            elements = [
+                element for element in result if element["category"] not in self.exclude
+            ]
+
+            return elements
+
         except requests.RequestException as req_err:
             # Handle any request-related exceptions
             print(f"Request Exception: {req_err}")
```

Comment:
FYI: print statements should generally not be used

Reply:
Thank you for notice, i will fix it